src/Repositories/PgSQLRepository.php

```diff
@@ -206,8 +206,8 @@ public function getProcedures(): Collection
             "SELECT proname, pg_get_functiondef(pg_proc.oid) AS definition
             FROM pg_catalog.pg_proc
                 JOIN pg_namespace ON pg_catalog.pg_proc.pronamespace = pg_namespace.oid
-            WHERE prokind = 'p'
-                AND pg_namespace.nspname = '$searchPath'"
+                JOIN pg_language ON pg_catalog.pg_proc.prolang = pg_language.oid
+            WHERE pg_language.lanname = 'sql' AND pg_namespace.nspname = '$searchPath' ORDER BY proname"
```

Comment:
A lot of unexpected procedures were generated, and the expected procedure is not generated.
Please revised and fix the query.

Test: https://github.com/kitloong/laravel-migrations-generator/actions/runs/4025538041/jobs/6936459002

Reply:
@kitloong 
May I know what version of pgsql you were using?

Reply:
https://github.com/kitloong/laravel-migrations-generator/blob/6.x/.github/workflows/tests.yml#L42

Reply:
Hi @ahmadsyamim, any update on this?